ArduCopter/mode_drift.cpp

```diff
@@ -26,14 +26,9 @@
  # define DRIFT_THR_MAX         0.787f  // throttle assist will be active when pilot's throttle is below this value
 #endif
 
-// drift_init - initialise drift controller
-bool Copter::ModeDrift::init(bool ignore_checks)
+void Copter::ModeDrift::enter()
 {
-    if (_copter.position_ok() || ignore_checks) {
-        return true;
-    }else{
-        return false;
-    }
+    // insert missing initialisations here
```

Comment:
Add a `TODO:` in beginning of comment? Easier to look up.

Reply:
Yep